build.gradle

```diff
@@ -783,11 +783,19 @@ project(':examples') {
 project(':clients') {
   archivesBaseName = "kafka-clients"
 
+  configurations {
+    jacksonDatabindConfig
+  }
+
+  conf2ScopeMappings.addMapping(1000, configurations.jacksonDatabindConfig, "provided")
```

Comment:
What does `1000` mean here? Probably worth adding a comment.

Reply:
Its a priority number used for comparison with the priority of other scopes. If multiple configurations are mapped with different scopes, then it will select scope with high priority. 
	 
It on our case we have only one configuration, So its not a issue. I just gave 1000 (highest priority) for the provided scope.